content/slides/en.yaml

```diff
@@ -60,6 +89,8 @@ levels:
 
                 `{print}` is used to show text on the screen.
             code: '{print} Show text on the screen'
+            notes: |-
```

Comment:
Maybe we can do a small recap of the first presentation by asking 'what will the output be' or 'what do you need to do to make the computer follow the instructions'